src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-form/service-form.component.html

```diff
@@ -395,6 +395,161 @@
           </div>
         </ng-container>
 
+        <!-- smb -->
+        <ng-container *ngIf="serviceForm.controls.service_type.value === 'smb'">
+          <div class="form-group row">
+            <label class="cd-col-form-label required"
+                   for="cluster_id"
+                   i18n>
+              Cluster id
+              <cd-helper>
+                <span>A short name identifying the SMB “cluster”. In this case a cluster is simply a management unit of one or more Samba services sharing a common configuration,
+                   and may not provide actual clustering or availability mechanisms.</span>
+              </cd-helper>
+            </label>
+            <div class="cd-col-form-input">
+              <input id="cluster_id"
+                     class="form-control"
+                     type="text"
+                     formControlName="cluster_id"
+                     placeholder="foo"
+                     i18n-placeholder>
+              <span class="invalid-feedback"
+                    *ngIf="serviceForm.showError('cluster_id', frm, 'required')"
+                    i18n>This field is required.</span>
+            </div>
+          </div>
+
+          <div class="form-group row">
+            <label class="cd-col-form-label required"
+                   for="config_uri">
+              <span i18n>Config URI</span>
+              <cd-helper i18n>
+                Configuration source that should be loaded by the samba-container as the primary configuration file.
+              </cd-helper>
+            </label>
+            <div class="cd-col-form-input">
+              <input id="config_uri"
+                     class="form-control"
+                     type="text"
+                     formControlName="config_uri"
+                     placeholder="rados://.smb/foo/scc.toml"
+                     i18n-placeholder>
+              <span class="invalid-feedback"
+                    *ngIf="serviceForm.showError('config_uri', frm, 'required')"
```

Comment:
good to have validation for url based on `Supported URI schemes include http:, https:, rados:, and rados:mon-config-key:.`